### PR TITLE
Add docs for HomePilot integration

### PR DESCRIPTION
**Pull request by ngres**
## Proposed change

This pull request adds the documentation for a new integration supporting [Rademacher HomePilot](https://www.rademacher.de/smart-home/produkte/homepilot) devices.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/48648
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2244
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @Nico0302, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#17248).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/homepilot.markdown [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Configuration
+
+To add the Rademacher HomePilot integration to your installation:
+
+- Go to the Integrations menu: **Configuration** >> **Integrations** in the UI.
+- Click the button with the `+` sign at the bottom right of the screen.
+- Select **Rademacher HomePilot** from the list of integrations.
+- Fill in the IP address or hostname of your Rademacher Hub or Bridge under **Host**.
+- If you protected your Rademacher Hub or Bridge with a password you should also enter it.
+- Click on **SUBMIT** to complete the setup.
```
Comment (frenck) [inline source/_integrations/homepilot.markdown]:
```suggestion
{% include integrations/config_flow.md %}
```

File: source/_integrations/homepilot.markdown [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+ha_category: Cover
+ha_iot_class: "Local Polling"
+ha_config_flow: true
+ha_codeowners:
+  - '@nico0302'
+ha_domain: homepilot
+---
+
+The [Rademacher HomePilot](https://www.rademacher.de/smart-home/produkte/homepilot) integration allows you to connect your Rademacher HomePilot Hub or Smart2Smart Bridge with Home Assistant.
+The [pyhomepilot library](https://github.com/Nico0302/pyhomepilot) is used for communicating.
```
Comment (frenck) [inline source/_integrations/homepilot.markdown]: The underlying does not add anything for the end-user and should not be part of the user documentation.

```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
```
Could not locate the included file 'integrations/config_flow.md' in any of ["/opt/build/repo/source/_includes"]. Ensure it exists in one of those directories and, if it is a symlink, does not point outside your site source. in /opt/build/repo/source/_integrations/homepilot.markdown
```

This indicates your base branch is out of date. Please perform a git rebase on the latest master to be up to date with upstream next.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**